conda_smithy/ci_skeleton.py

```diff
@@ -0,0 +1,42 @@
+"""Generate skeleton for using conda-forge CI configuration files outside of
+a feedstock (i.e. a outside of a forge context). This enables people to use
+conda-forge to run their CI without having to worry about how to set it up,
```

Comment:
I would maybe change the wording of this, to me it sounds like conda-forge is supplying CI resources (conda-forge to run their CI). My understanding is that we are providing our CI configurations so others can instantiate their own CI workflows using the conda-forge templates and configuration infrastructure.